can/interface.py

```diff
@@ -29,7 +29,9 @@ def _get_class_for_interface(interface):
     try:
         module_name, class_name = BACKENDS[interface]
     except KeyError:
-        raise NotImplementedError("CAN interface '{}' not supported".format(interface))
+        raise NotImplementedError(
+            "CAN interface '{}' not supported".format(interface)
+        ) from None
```

Comment:
raise from KeyError reference

Reply:
Same as above. This was changed to please pylint (I think it was that one).